utils/R/utils.R

```diff
@@ -618,9 +618,9 @@ load.modelpkg <- function(model){
 ##' @author Istem Fer
 misc.convert <- function(x, u1, u2){
   if(u1 == "umol C m-2 s-1" & u2 == "kg C m-2 s-1"){
-    val <- ud.convert(x, "ug m-2 s-1", "kg ha-1 yr-1") * 12 
+    val <- x / (1e09) * 12 
```

Comment:
That's a good idea, I changed it to udunits, there is a package called `OrgMassSpecR` which has functions that can give you atomic masses for molecules/atoms but not sure whether it's worth using it here